### PR TITLE
Add simple build tests for usbarmory bits

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: integration/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
+FROM ubuntu as testbase
 
 WORKDIR /testbase
 
 ARG GOFLAGS=""
 ENV GOFLAGS=$GOFLAGS
 ENV GO111MODULE=on
 
-RUN apt-get update && apt-get -y install curl unzip wget default-mysql-client docker-compose lsof xxd
+RUN apt-get update && apt-get -y install curl default-mysql-client docker-compose lsof unzip wget xxd
```
Comment (mhutchinson) [inline integration/Dockerfile:12]: I wonder what we're using the mysql-client for?

File: integration/Dockerfile [hunk 2]
```diff
@@ -1,23 +1,30 @@
 # This Dockerfile builds a base image for the CloudBuild integration testing.
 # It just extends a golang/linux base image with a few extra tools which the
```
Comment (mhutchinson) [inline integration/Dockerfile:2]: You can drop the golang part of this line now.

**Review by mhutchinson — APPROVED** [review]

File: integration/Dockerfile (hunk 1 above)
Comment (AlCutter) [reply integration/Dockerfile:12]: Ah, it used to be for the resetDB stuff, but I think that's gone now.

**Pull request merged**